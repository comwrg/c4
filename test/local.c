#include <stdio.h>

int main() {
    int date = 522;
    printf("%d\n", date);
    return 2333;
}
