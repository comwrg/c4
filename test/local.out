522
exit(2333)
